SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1647,10 +1647,18 @@ void USpatialNetDriver::PollPendingLoads()
 
 void USpatialNetDriver::TickFlush(float DeltaTime)
 {
-	PollPendingLoads();
-
 	const USpatialGDKSettings* SpatialGDKSettings = GetDefault<USpatialGDKSettings>();
 
+	if (SpatialGDKSettings->bRunSpatialWorkerConnectionOnGameThread)
```

Comment:
Minor - would rather that only the worker connection had to care about this flag. Then this is all hidden in the implementation details on the worker connection stuff to poll ops or send messages.

Reply:
Okay - I'll make that change

Reply:
Discussed offline, will hold off for now as this setting is going to be temporary anyway.